src/main/java/org/jnode/fs/xfs/superblock/MiscFlags.java

```diff
@@ -0,0 +1,37 @@
+package org.jnode.fs.xfs.superblock;
+
+import java.util.List;
+
+import org.jnode.fs.xfs.Flags;
+
+/**
+ * Miscellaneous flags.
+ * Flags from the sb_flags.
+ *
+ * @see <a href="https://github.com/torvalds/linux/blob/b2a88c212e652e94f1e4b635910972ac57ba4e97/fs/xfs/libxfs/xfs_format.h#L273">xfs_format.h</a>
+ */
+public enum MiscFlags implements Flags {
+    /**
+     * no flags set.
+     */
+    NOFLAGS(0x00),
```

Comment:
I don't think you need this value.
... also after digging down to `FlagUtil.isSet()` this will be set for any value, which doesn't make sense eg.
```
FlagUtil.isSet(6) { return (0 & 6) == 0; }
```

Reply:
The `Table 18.1` in the document does mention something that we may want to support the 0. I'll change FlagUtil to support it.

Reply:
Thanks, I also added the 0 check in FlagUtil.